pkg/metrics/metrics.jsx

```diff
@@ -975,66 +979,47 @@ class MetricsMinute extends React.Component {
-                    {this.state.expanded && this.state.logsUrl &&
-                        <Button variant="link" isInline onClick={e => cockpit.jump(this.state.logsUrl)}>
-                            { _("View detailed logs") }
-                        </Button>}
-                </div>
-            </div>;
 
-            let body = " "; // Cannot be false-y, otherwise table does not show '>'
-            if (this.state.expanded) {
-                body = <div className="cockpit-log-panel">
-                    {this.state.logs === null
-                        ? _("Loading...")
-                        : this.state.logs.length === 0
-                            ? <span className="pf-u-py-sm">{ _("No logs found") }</span>
-                            : this.state.logs
-                    }
-                </div>;
-            }
+            const logsPanel = (
+                <>
+                    {(this.state.logs?.length && this.state.logsUrl) && <Button variant="secondary" onClick={e => cockpit.jump(this.state.logsUrl)}>{_("View all logs")}</Button>}
+                    <div className="cockpit-log-panel">
+                        {this.state.logs?.length ? this.state.logs : _("No log entries")}
+                    </div>
+                </>
+            );
 
-            const entry = [{
-                props: { key: timestamp, 'data-row-id': timestamp },
-                columns: [{ title: desc }],
-                hasPadding: false,
-                expandedContent: body,
-            }];
-
-            events = <div className="metrics-events-wrapper">
-                <ListingTable aria-label={ _("Event logs") }
-                                      className="metrics-events"
-                                      style={{ "--pf-c-table--BorderColor": "#fff" }}
-                                      showHeader={false}
-                                      variant="compact"
-                                      afterToggle={this.expand}
-                                      gridBreakPoint=''
-                                      columns={[
-                                          { title: _("Event") },
-                                      ]}
-                                      rows={entry} />
+            desc = <div className="metrics-events">
+                <time>{ timeformat.time(timestamp) }</time>
+                <span className="spikes_count" />
+                <Popover position="right" hasAutoWidth className="metrics-events-popover" bodyContent={logsPanel}>
+                    <Button
+                        variant="link" isInline
+                        className="spikes_info"
+                        onClick={() => this.expand(!this.state.expanded)}>
+                        <span className="type">
+                            { this.props.events.events.map(t => RESOURCES[t].event_description).join(", ") }
+                        </span>
+                    </Button>
```

Comment:
I'm not sure how exactly this is buggy, but if there is more than one spike type in a second, then it's not a link:

![image](https://user-images.githubusercontent.com/200109/224911432-42c17fba-7629-4347-abb2-fb78c9e7cea6.png)

I.e. I can click on the first "Disk I/O" and "CPU"  to get the journal popup for 7:02 and 7:08. But the "CPU, Swap, Network I/O" text is not a link, and not clickable to get the 7:07 journal.

This is repeatable: If I cause two spikes in the same minute, vs. causing a single spike.

```
nc -n -vv -l 2000 > /dev/null & sleep 1; nc -vv localhost 2000 </dev/zero
echo 3 > /proc/sys/vm/drop_caches; cat /dev/vda > /dev/null
dd if=/dev/urandom of=/dev/null
```

Reply:
Hmm, this feels a bit funny. After letting it run for a bit longer, my newer "not a link" suddenly turned out to be a link. I now saw the extra commit to not make it a link if there are no log messages. `journalctl -b | grep 06:07` indeed shows no logs, and the previously missing link for 07:15 now exists:

![image](https://user-images.githubusercontent.com/200109/224913748-8a1a2c1b-8099-4d6f-af68-be4cfe1ce731.png)

So this is just some journal loading lag, unrelated to this PR and most probably just a bit wonky for the current minute.